### DISCUSSION TITLE
Is uv.lock supported by lockFileMaintenance?

### DISCUSSION THREAD

**Discussion by Adirelle** [renovatebot/renovate#32833] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitLab, 39.42.2

### Please tell us more about your question or problem

According to [Renovate's documentation](https://docs.renovatebot.com/configuration-options/#lockfilemaintenance), the lockfile of uv, `uv.lock`, does not seem to be taken into account by `lockfilemaintenance`. It does not appear in the dependecency dashboard either, but I do not know if it is to be expected.

However, I might have overlooked something, so: is uv.lock supported by lockFileMaintenance?

**Comment by mkniewallner** [comment, marked as answer]
It is supported yes (for instance https://github.com/fpgmaas/deptry/pull/963).

**Reply by Adirelle** [reply]
I see. Thank you. The documentation could be updated I guess.

**Reply by rarkins** [reply]
PR welcome
